.github/workflows/windows.yml

```diff
@@ -323,7 +323,7 @@ jobs:
           }
       -
         name: Starting containerd
-        if: matrix.runtime == 'containerd'
+        if: matrix.runtime == 'containerd' && matrix.os == 'windows-2019'
```

Comment:
I don't know GitHub Actions that well, but would it be clearer to have the `runtime` axis with three values (`builtin`, `managed-containerd`, `system-containerd`) and also flag at that point to skip running `windows-2019/system-containerd` and `windows-2022/managed-containerd`? That will also make it easier to adjust for "Windows 2025" for example. (Playing the long game. ^_^)

Reply:
Ah yes, there's [an `exclude` parameter for the matrix definition](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations)..

Reply:
Using `include` pattern in the matrix to run specific against specific fields would also work.

Reply:
Ok. However I think that there is other open questions here and https://github.com/moby/moby/issues/43892 to be answered first before I will continue this further.